kuka_kr210_support/urdf/kr210r3100.xacro

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" ?>
+<robot name="kuka_kr210" xmlns:xacro="http://wiki.ros.org/xacro">
+  <xacro:arg name="prefix" default="" />
+
+  <xacro:include filename="$(find kuka_kr210_support)/urdf/kr210r3100_macro.xacro"/>
+
+  <xacro:kuka_kr210r3100 prefix=""/>
```

Comment:
should the prefix arg be used here?
<xacro:kuka_kr210r3100 prefix="$(arg prefix)"/>

Reply:
```suggestion
  <xacro:kuka_kr210r3100 prefix="$(arg prefix)"/>
```